pennylane/transforms/__init__.py

```diff
@@ -74,6 +74,16 @@
     ~transforms.CommutationDAG
     ~transforms.CommutationDAGNode
 
+Transform for Clifford+T compilation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following function assist in decomposing operations to the Clifford+T basis.
```

Comment:
```suggestion
The following functions assist in decomposing operations to the Clifford+T basis.
```
I know it's just one for now, but more will come.

Reply:
CC @rmoyard - we will be tidying up the transforms module, so this will eventually be moved.